src/EditorFeatures/Core/Implementation/ForegroundNotification/ForegroundNotificationService.cs

```diff
@@ -66,6 +66,8 @@ public void RegisterNotification(Func<bool> action, int delay, IAsyncToken async
             _workQueue.Enqueue(new PendingWork(current + delay, action, asyncToken, cancellationToken));
         }
 
+        internal void ReleaseCancelledItems() => _workQueue.ReleaseCancelledItems();
```

Comment:
who calls this?

Reply:
:arrow_right: For now the only caller is test code that's not yet submitted in a PR